### ISSUE TITLE
Inconsistent styling in results window

### ISSUE THREAD

**Issue by chlowell** [microsoft/dotnet-apiport#780]
In the Portability Analysis Results window, only the first entry's buttons have the correct style:

![image](https://user-images.githubusercontent.com/10964656/60845831-5a12f600-a192-11e9-8ea1-2b9ee8d41ea9.png)
(this screenshot shows what it looks like with High Contrast Black theme)

The inconsistency also creates an accessibility issue: in the second and third rows, the contrast ratio between the buttons and their labels is too low.

**Comment by chlowell** [comment]
Further investigation reveals each row is styled when it is created, and the style applied is correct for the state of Windows' high contrast theme when the row is created. The inconsistency I observed is due to toggling high contrast between analyses.
